api/error_injection.cc

```diff
@@ -24,10 +26,26 @@ void set_error_injection(http_context& ctx, routes& r) {
     hf::enable_injection.set(r, [](std::unique_ptr<request> req) {
         sstring injection = req->param["injection"];
         bool one_shot = req->get_query_param("one_shot") == "True";
-        auto& errinj = utils::get_local_injector();
-        return errinj.enable_on_all(injection, one_shot).then([] {
-            return make_ready_future<json::json_return_type>(json::json_void());
-        });
+        auto params = req->content;
+
+        if (params.size() > 1024 * 1024) {
+            // This is a hard limit, because we don't want to allocate
+            // too much memory or block the thread for too long.
+            throw httpd::bad_param_exception(fmt::format("Injection parameters are too long, max length is {}", 1024 * 1024));
```

Comment:
The magic constant is used twice, so it makes sense to make a const var out of it.

Reply:
Also, why use `fmt::format` when you use `format` below? Please stick to a single version